tests/components/myuplink/conftest.py

```diff
@@ -64,34 +63,109 @@ async def setup_credentials(hass: HomeAssistant) -> None:
     )
 
 
+# Fixture group for device API endpoint.
+
+
+@pytest.fixture(scope="session")
+def load_device_file():
+    """Load fixture file for device endpoint."""
+    return load_json_value_fixture("device.json", DOMAIN)
+
+
 @pytest.fixture
-def mock_myuplink_client() -> Generator[MagicMock, None, None]:
-    """Mock a myuplink client."""
+def device_fixture(load_device_file):
+    """Fixture for device."""
+    return Device(load_device_file)
```

Comment:
Import from`json_loads` from `homeassistant.util.json`.
```suggestion
def device_fixture(load_device_file) -> Device:
    """Fixture for device."""
    return Device(json_loads(load_device_file))
```

Reply:
OK